main.tf

```diff
@@ -34,6 +34,6 @@ resource "null_resource" "save_ssh_keys" {
   }
 
   provisioner "local-exec" {
-    command = "echo \"${tls_private_key.default.private_key_pem}\" > ${var.ssh_public_key_path}/${module.label.id} && chmod 600 ${var.ssh_public_key_path}/${module.label.id}"
+    command = "echo \"${tls_private_key.default.private_key_pem}\" > ${var.ssh_public_key_path}/${module.label.id}.pem && chmod 600 ${var.ssh_public_key_path}/${module.label.id}.pem"
```

Comment:
I propose we add a parameter called `private_key_extension` and default it to `""` for backwards compatibility. I think that satisfies both requirements. @comeanother @SweetOps any thoughts?

Reply:
e.g. I've used the `id_rsa` extension in the past.

Reply:
Great. I'll create the parameter and update the pull request. Thanks.